src/daemon/tabswitcher.vala

```diff
@@ -255,6 +256,16 @@ namespace Budgie {
 			return pos1 < pos2 ? -1 : 1;
 		}
 
+		private uint get_preferred_max_children() {
```

Comment:
The name of this function is misleading. Maybe `get_visible_children` instead, or something?

Reply:
True

Reply:
Resolved with eed33e85a9dd428c8c9a2c6e43ceafc1777abfae